src/components/v5/common/ActionSidebar/partials/UserSelect/UserSelect.tsx

```diff
@@ -171,8 +173,8 @@ const UserSelect: FC<UserSelectProps> = ({
 
           //   return triggerRef.querySelector(`.${LABEL_CLASSNAME}`);
           // }}
-          trigger={!isUserAddressValid ? 'hover' : undefined}
-          placement={isUserAddressValid ? 'top' : 'bottom'}
+          trigger={isUserAddressInvalid ? 'hover' : undefined}
```

Comment:
I don't think this line actually does anything as the default trigger type for the tooltip component is hover anyway - so this entire line can be removed.